test/yz_dt_extractor_tests.erl

```diff
@@ -82,4 +90,10 @@ raw_type(set) ->
       );
 raw_type(counter) ->
     ?COUNTER_TYPE(
-       element(2,?COUNTER_TYPE:update({increment, 10}, <<0>>, ?COUNTER_TYPE:new()))).
+       element(2,?COUNTER_TYPE:update({increment, 10}, <<0>>, ?COUNTER_TYPE:new())));
+raw_type(hll) ->
+    ?HLL_TYPE(
+       element(2,?HLL_TYPE:update({add_all, [<<"T">>, <<"h">>, <<"r">>, <<"i">>,
+                                             <<"l">>, <<"l">>, <<"a">>, <<"s">>,
+                                             <<"f">>, <<"u">>, <<"h">>]},
```

Comment:
@jvoegele just using unique (and non-unique) simple binaries for the test. See the <<"l">> for example. It also spells "Thrill as fuh" and is a play on Andy Pavlo's Trill as fuck post... jokes are better explained.